jwst/associations/lib/dms_base.py

```diff
@@ -43,30 +43,30 @@
     'mir_flatmrs':       'flat',
     'mir_flatimage-ext': 'flat',
     'mir_flatmrs-ext':   'flat',
-    'mir_tacq':          'target_acquistion',
+    'mir_tacq':          'target_acquisition',
     'nis_dark':          'dark',
     'nis_focus':         'engineering',
     'nis_lamp':          'engineering',
-    'nis_tacq':          'target_acquistion',
-    'nis_taconfirm':     'target_acquistion',
+    'nis_tacq':          'target_acquisition',
+    'nis_taconfirm':     'target_acquisition',
     'nrc_dark':          'dark',
     'nrc_flat':          'flat',
     'nrc_focus':         'engineering',
     'nrc_led':           'engineering',
-    'nrc_tacq':          'target_acquistion',
-    'nrc_taconfirm':     'target_acquistion',
+    'nrc_tacq':          'target_acquisition',
+    'nrc_taconfirm':     'target_acquisition',
     'nrs_autoflat':      'autoflat',
     'nrs_autowave':      'autowave',
-    'nrs_confirm':       'target_acquistion',
+    'nrs_confirm':       'target_acquisition',
     'nrs_dark':          'dark',
     'nrs_focus':         'engineering',
     'nrs_image':         'engineering',
```

Comment:
NIRSpec imaging exposures are supposed to receive normal ``calwebb_image2`` processing and hence need to be included as science members in a level-2b ASN. What effect will this value of "engineering" have on them?

Reply:
Part 1 of answer: Will need to dig up the conversations, but why NRS_IMAGE is marked `engineering` was because it would not be used for science. If that is not the case, this will be fixed.

Regardless of Part 1, the answer to the question is Part 2: See the next comment...